docs/UserGuide.md

```diff
@@ -140,13 +140,8 @@ Format: `add n/NAME p/PHONE_NUMBER [tg/TELEGRAM] [gh/GITHUB] [e/EMAIL] [a/ADDRES
 
 * `NAME` should only contain alphanumeric characters and spaces. It should not be blank and be at most 30 characters long.
 * `PHONE_NUMBER` should only contain numeric characters, and should be 3-15 digits long.
-* `TELEGRAM`, if specified, should only contain alphanumeric characters and underscore (_), and should be 5-40 characters long. It should not be blank and should not have multiple consecutive underscores or begin or end with an underscore.
-* `GITHUB`, if specified, should only contain alphanumeric characters and hyphen (-), and be at most 39 characters long. It should not be blank and should not have multiple consecutive hyphens or begin or end with a hyphen.
-* `ADDRESS`, if specified, should only contain alphanumeric characters, spaces, and the following punctuations: **!"#$&'()\*+,-.:;<=>?@**. It should not be blank and be at most 80 characters long.
-* `EMAIL`, if specified, should be a **valid** email with length not exceeding 60 characters.
-* **Valid** `EMAIL` are emails that are of the format *local-part@domain* and adhere to the following constraints:
-  * The local-part should only contain alphanumeric characters and the following special characters: **+_.-**. It cannot start or end with any special characters and cannot have consecutive special characters.
-  * The domain part is made up of domain labels separated by periods (e.g. domain-label1.domain-label2.com). It must end with a domain label at least 2 characters long, each domain label starts and end with alphanumeric characters, and each domain label consists of only alphanumeric characters, separated only by hyphens, if any.
+* `EMAIL`, if specified, should be a **valid** email.
+* Similarly, `TELEGRAM` and `GITHUB`, if specified, should be **valid** usernames as well.
```

Comment:
I think can dont write this at all.

Reply:
i was thinking about that, but that means name and phone number shouldn't be mentioned at all also right? As all attributes should be equivalent, so if one is mentioned, the rest should be also?

Reply:
Yup, I believe we can remove the name and phone ones also. Same for some of the constraints for addEvent and editEvent